packages/react-components/react-drawer/library/src/components/OverlayDrawer/useOverlayDrawerStyles.styles.ts

```diff
@@ -47,7 +47,7 @@ export const useOverlayDrawerStyles_unstable = (state: OverlayDrawerState): Over
   const resetStyles = useDrawerResetStyles();
   const rootStyles = useDrawerRootStyles();
 
-  const absoluteStyles = !!state.mountNode && rootStyles.absolute;
+  const absoluteStyles = !!state.hasMountNodeElement && rootStyles.absolute;
```

Comment:
nit: we might not need to do type coercion because `state.hasMountNodeElement` is always a boolean value. 

```suggestion
  const absoluteStyles = state.hasMountNodeElement && rootStyles.absolute;
```